consensus/src/round_manager.rs

```diff
@@ -1222,6 +1224,47 @@ impl RoundManager {
         result
     }
 
+    async fn new_qc_from_order_vote_msg(
+        &mut self,
+        order_vote_msg: &OrderVoteMsg,
+    ) -> anyhow::Result<()> {
+        ensure!(
```

Comment:
this is already checked in the verify function

Reply:
Removed the duplicate check.